docs/arguments.md

```diff
@@ -257,6 +257,10 @@ Environment Variable: WATCHTOWER_HTTP_API_TOKEN
 ```
 
 ## HTTP API periodic polls
+
+!!! warning
+This argument is not currently available in the 1.3.0 release.
+
```

Comment:
The plan is to release today, so this won't be deployed until the update anyway.

```suggestion
```

Reply:
Ah OK, would you like me to remove this commit?

Reply:
No worries, I'll take care of it as we're releasing today :)